include/seqan3/range/views/minimiser_hash.hpp

```diff
@@ -0,0 +1,148 @@
+ * it is highly likely that the minimiser will start with a repetition of A’s as well. Because it is only one A shorter,
+ * depending on how long the repetition is this might go on for multiple window shifts. Saving these only slightly
+ * different minimiser makes no sense because they contain no new information about the underlying sequence plus
+ * sequences with a repetition of A’s will be seen as more similar to each other than they actually are.
+ * As [Marçais et al.](https://doi.org/10.1093/bioinformatics/btx235) have shown, randomizing the order of the k-mers
+ * can solve this problem. Therefore, a random seed is used as a default to XOR all k-mers, thereby randomzing the
+ * order. The user can change the seed to any other value he or she thinks is useful. A seed of 0 is returning the
+ * lexicographical order.
+ *
+ * \attention
+ * As f the the seqan3::views::kmer_hash the alphabet size \f$\sigma\f$ of the alphabet of `urange` and the number of
+ * 1s \f$s\f$ of `shape` it must hold that \f$s>\frac{64}{\log_2\sigma}\f$, i.e. hashes resulting from the
+ * shape/alphabet combination can be represented in an `uint64_t`.
+ *
+ * ### View properties
+ *
+ * | Concepts and traits              | `urng_t` (underlying range type)   | `rrng_t` (returned range type)   |
+ * |----------------------------------|:----------------------------------:|:--------------------------------:|
+ * | std::ranges::input_range         | *required*                         | *preserved*                      |
+ * | std::ranges::forward_range       | *required*                         | *preserved*                      |
+ * | std::ranges::bidirectional_range |                                    | *lost*                           |
+ * | std::ranges::random_access_range |                                    | *lost*                           |
+ * | std::ranges::contiguous_range    |                                    | *lost*                           |
+ * |                                  |                                    |                                  |
+ * | std::ranges::viewable_range      | *required*                         | *guaranteed*                     |
+ * | std::ranges::view                |                                    | *guaranteed*                     |
+ * | std::ranges::sized_range         |                                    | *lost*                           |
+ * | std::ranges::common_range        |                                    | *lost*                           |
+ * | std::ranges::output_range        |                                    | *lost*                           |
+ * | seqan3::const_iterable_range     |                                    | *preserved*                      |
+ * |                                  |                                    |                                  |
+ * | std::ranges::range_reference_t   | seqan3::semialphabet               | std::integral                    |
```

Comment:
```suggestion
 * | std::ranges::range_reference_t   | seqan3::semialphabet               | std::size_t                    |
```

It should just be the value_type of the kmer_hash view?